test/functional/tools/samples_tool_conf.xml

```diff
@@ -44,8 +44,11 @@
   <tool file="inputs_as_json_profile.xml" />
   <tool file="inputs_as_json_with_paths.xml" />
   <tool file="filter_static_regexp.xml" />
+  <tool file="select_from_dataset.xml" />
   <tool file="dbkey_filter_input.xml" />
   <tool file="dbkey_filter_multi_input.xml" />
+  <tool file="dbkey_filter_collection.xml" />
+  <tool file="dbkey_filter_paired_collection.xml" />
```

Comment:
@bernt-matthias It was a while ago, but did you mean to add a `dbkey_filter_paired_collection.xml` test tool? Or should we remove this entry?

Reply:
@nsoranzo I guess I wanted to add it, but forgot to add the actual test tool. I guess for now its best to remove the entry from samples_tool_conf.xml .

Reply:
OK, thanks! Done in https://github.com/galaxyproject/galaxy/pull/13035